Libs/GObject/Classes/Object.cs

```diff
@@ -26,7 +26,7 @@ public partial class Object : INotifyPropertyChanged, IDisposable
 
         #region Properties
 
-        protected internal IntPtr Handle { get; private set; }
+        public IntPtr Handle { get; private set; }
```

Comment:
Use GetHandle instead of making Handle public

Reply:
Fixed